matt-collection.sql

```diff
@@ -0,0 +1,68 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 5446
+#
+# https://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.33)
+# Database: matt-collection
+# Generation Time: 2021-03-29 10:54:42 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+SET NAMES utf8mb4;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table games
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `games`;
+
+CREATE TABLE `games` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `bgg-id` int(11) unsigned NOT NULL,
+  `name` varchar(80) NOT NULL DEFAULT '',
```

Comment:
up the varchar to 255 (its a good default, any less can sometimes get into trouble)